Assets/MRTK/Providers/XRSDK/XRSDKBoundarySystem.cs

```diff
@@ -34,12 +45,7 @@ public class XRSDKBoundarySystem : BaseBoundarySystem
         /// <inheritdoc/>
         public override void Initialize()
         {
-            if (!Application.isPlaying) { return; }
-
-            List<InputDevice> devices = new List<InputDevice>();
-            InputDevices.GetDevicesWithCharacteristics(InputDeviceCharacteristics.HeadMounted, devices);
-
-            if (devices.Count <= 0) { return; }
+            if (!Application.isPlaying || !IsXRDevicePresent) { return; }
```

Comment:
I think that since we now do this exact check in the base class, we can remove the whole overridden `Initialize` from this class like you did the other one. That way they align again in behavior as well.